index.html

```diff
@@ -4628,15 +4628,20 @@ <h2>Equivalence Properties</h2>
     <section>
       <h2>Content Integrity Protection</h2>
       <p>
-<a>DID documents</a> which include external JSON-LD contexts (see
-<a href="#json-ld"></a>) or any other links to external machine-readable
-content are vulnerable to tampering.
-      </p>
-      <p>
-<a>DID document</a> consumers can cache local static copies of JSON-LD contexts
-and/or verify the integrity of external contexts against the cryptographic
-hash for the context as registered in the DID Specification Registries
-(see the registration process for more detail) [[?DID-SPEC-REGISTRIES]].
+<a>DID documents</a> which include links to external machine-readable content
+such as images, web pages, or schemas are vulnerable to tampering. It is
+strongly advised that external links are integrity protected using solutions
+such as a hashlink [[?HASHLINK]]. External links are to be avoided if they
+cannot be integrity protected and the <a>DID document</a>'s integrity is
+dependent on the external link.
+      </p>
+      <p>
+One example of an external link where the integrity of the <a>DID document</a>
+itself could be affected is the JSON-LD Context [[JSON-LD]]. To protect against
```

Comment:
```suggestion
values and semantics could be affected by the JSON-LD Context [[JSON-LD]] depending on the JSON-LD implementation. To protect against
```

Reply:
Looks good, just figured I'd add a bit more detail to why the integrity protection matters. I don't think this suggestion can be merged directly though because the line length is likely off now.

Reply:
Hmm, this statement isn't accurate. The JSON-LD implementation isn't typically in charge of ensuring the integrity of the context... that is typically delegated to the document loader / caching layer / application. I'm going to reject the change request on those grounds.